projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-upload-media-v1-1-endpoint.php

```diff
@@ -93,6 +93,19 @@ public function callback( $path = '', $blog_id = 0 ) {
 		// We're splitting out videos for Jetpack sites.
 		foreach ( $media_files as $media_item ) {
 			if ( preg_match( '@^video/@', $media_item['type'] ) && $is_jetpack_site ) {
+				if ( defined( 'IS_WPCOM' ) &&
+					IS_WPCOM &&
+					defined( 'VIDEOPRESS_JETPACK_VIDEO_ENABLED' ) &&
+					VIDEOPRESS_JETPACK_VIDEO_ENABLED ) {
```

Comment:
Nitpicking, feel free to ignore.

If possible, I'd find it easier to read like this
```suggestion
				if ( defined( 'IS_WPCOM' ) && IS_WPCOM &&
					defined( 'VIDEOPRESS_JETPACK_VIDEO_ENABLED' ) && VIDEOPRESS_JETPACK_VIDEO_ENABLED
				) {
```
Or at least put the closing `) {` on a new line to visually separate the condition from the body, like this
```suggestion
				if ( defined( 'IS_WPCOM' ) &&
					IS_WPCOM &&
					defined( 'VIDEOPRESS_JETPACK_VIDEO_ENABLED' ) &&
					VIDEOPRESS_JETPACK_VIDEO_ENABLED
				) {
```